pwiz_tools/Skyline/Model/Lib/EncylopeDiaLibrary.cs

```diff
@@ -769,29 +820,69 @@ public static EncyclopeDiaLibrary Deserialize(XmlReader reader)
         }
 
         private static ElibSpectrumInfo MakeSpectrumInfo(PeptideLibraryKey libraryKey,
-            IDictionary<string, ScoredFileData> fileDatas, IDictionary<string, int> sourceFileIds)
+            IDictionary<string, FileData> fileDatas, IDictionary<string, int> sourceFileIds)
         {
             double bestScore = double.MaxValue;
             string bestFileName = null;
 
             foreach (var entry in fileDatas)
             {
-                if (!entry.Value.Score.HasValue || !entry.Value.FileData.ApexTime.HasValue)
+                if (!double.IsNaN(entry.Value.PeakBounds.Score) || !entry.Value.ApexTime.HasValue)
                 {
                     continue;
                 }
-                if (bestFileName == null || entry.Value.Score < bestScore)
+                if (bestFileName == null || entry.Value.PeakBounds.Score < bestScore)
                 {
                     bestFileName = entry.Key;
-                    bestScore = entry.Value.Score.Value;
+                    bestScore = entry.Value.PeakBounds.Score;
                 }
             }
             return new ElibSpectrumInfo(libraryKey, bestFileName == null ? -1 : sourceFileIds[bestFileName],
                 fileDatas.Select(
-                    entry => new KeyValuePair<int, FileData>(sourceFileIds[entry.Key], entry.Value.FileData)));
+                    entry => new KeyValuePair<int, FileData>(sourceFileIds[entry.Key], entry.Value)));
 
         }
 
+        [XmlRoot("encyclopedia_spectrum_info")]
+        public sealed class ElibSpectrumHeaderInfo : SpectrumHeaderInfo
+        {
+            public ElibSpectrumHeaderInfo(string libraryName, double? score)
+                : base(libraryName, score, BiblioSpec.ScoreType.GenericQValue.NameInvariant)
+            {
+            }
+
+            public override string LibraryTypeName => @"EncyclopeDIA";
+
+            public override IEnumerable<KeyValuePair<PeptideRankId, string>> RankValues
+            {
+                get { yield break; }
+            }
+
+            #region Implementation of IXmlSerializable
+
+            /// <summary>
+            /// For serialization
+            /// </summary>
+            private ElibSpectrumHeaderInfo()
+            {
+            }
+
+            // ReSharper disable once MemberHidesStaticFromOuterClass
+            public static ElibSpectrumHeaderInfo Deserialize(XmlReader reader)
+            {
+                return reader.Deserialize(new ElibSpectrumHeaderInfo());
+            }
+
+            public override void ReadXml(XmlReader reader)
+            {
+                // Consume tag

```

Comment:
I think the "Consume tag" comment belongs on the next line because it's referring to the "reader.Read()" call, not the "base.ReadXml(reader)" call.